src/DB/index.ts

```diff
@@ -0,0 +1,16 @@
+// import { Pool } from "pg";
```

Comment:
why is this commented out? I like the idea of having a connection pool

Reply:
i was having trouble destructuring and passing my env variables instead of having to write my database information there